spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -63,7 +63,7 @@ sealed class SpdxExpression {
          */
         @JsonCreator
         @JvmStatic
-        fun parse(expression: String) = parse(expression, Strictness.ALLOW_ANY)
+        fun parse(expression: String): SpdxExpression = parse(expression, Strictness.ALLOW_ANY)
```

Comment:
Please add a rationale to the commit message.

Reply:
Done.

Reply:
Commit message: "*To* increase [...]". That's also a highly subjective "rationale", esp. as this 1. is a one-liner, and 2. it calls a function of the same name which is defined directly below, *including* the return type.

Reply:
> That's also a highly subjective "rationale", esp. as this 1. is a one-liner, and 2. it calls a function of the same name which is defined directly below, including the return type.

That comment implies that it does not increase readability because of the function below and this in fact is highly subjective as well. So what's you point - want me to scratch that change ?

Reply:
I haven't used 'To increase [...]' because I got complaints about writing this kind of commit message in the past because the message is not a full sentence.

Reply:
> I got complaints about writing this kind of commit message in the past because the message is not a full sentence.

But "Increase the readability" is not a full sentence either, so I do not get your point.

> So what's you point - want me to scratch that change ?

I was trying to imply that the *rationale* needs improvement, not the *change*. A good (and objective) rationale would have been: "This addresses the IDE hint to explicitly add the return type to public functions for API stability."

Reply:
I still think this commit didn't need a rationale in the first place because it is common sense which again is subjective is seems.

Reply:
That's how it is when working in a distributed team: Not only opinions differ, but also contexts are different. In the context of the changes you're currently working on it might be obvious for *you* that this change makes sense, but this is not necessarily the case for others.

Also, we demand in our existing [contribution guidelines](https://github.com/oss-review-toolkit/ort/blob/master/CONTRIBUTING.md) to "provide a rationale explaining *why* you believe your change is necessary". If we ask others to do that, we should be a good example and do it all the more.

Reply:
You are right, thanks for the discussion.

Reply:
+1 for "To increase readability." If you check the Git log we already have a lot of similar commit messages.